<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DriveControlPanel</class>
 <widget class="QMainWindow" name="DriveControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control Panel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>411</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="regular">
     <widget class="QPushButton" name="regularDriveUp_button">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>240</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
     <widget class="QSlider" name="regularServo_slider">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-45</number>
      </property>
      <property name="maximum">
       <number>45</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="regularServoValue_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="regularServoSubmit_button">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>160</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="regularDriveRight_button">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>300</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Right</string>
      </property>
     </widget>
     <widget class="QPushButton" name="regularDriveDown_button">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>360</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
     <widget class="QPushButton" name="regularDriveLeft_button">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>300</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Left</string>
      </property>
     </widget>
     <widget class="QLabel" name="regularMode_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Regular Mode</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="regularServo_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Servo Position</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="regularDrive_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Drive Buttons</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tank">
     <widget class="QLabel" name="tankMode_label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tank Mode</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QRadioButton" name="tankServo45_radio">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>45</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">tankServoGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="tankServo90_radio">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>100</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>90</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">tankServoGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="tankServo0_radio">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">tankServoGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="tankDown_button">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>290</y>
        <width>91</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tankUp_button">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>91</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
     <widget class="QLabel" name="tankDrive_label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Drive Buttons</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="tankServo_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Servo Position</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="custom">
     <widget class="QLabel" name="customMode_label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Custom Mode</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="customLeftUp_button">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>290</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>↑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="customUpDown_button">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>340</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>↑↓</string>
      </property>
     </widget>
     <widget class="QPushButton" name="customLeftDown_button">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>390</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>↓</string>
      </property>
     </widget>
     <widget class="QPushButton" name="customDownDown_button">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>390</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>↓↓</string>
      </property>
     </widget>
     <widget class="QPushButton" name="customRightDown_button">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>390</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>↓</string>
      </property>
     </widget>
     <widget class="QPushButton" name="customDownUp_button">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>340</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>↓↑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="customRightUp_button">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>290</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>↑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="customUpUp_button">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>290</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>↑↑</string>
      </property>
     </widget>
     <widget class="QLabel" name="customServo_label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Servo Position</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="customDrive_label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>260</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Drive Control</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="customServoSubmit_button">
      <property name="geometry">
       <rect>
        <x>169</x>
        <y>210</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="customServoLeftFront_edit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="customServoLeftMiddle_edit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="customServoLeftBack_edit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="customServoRightMiddle_edit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>120</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="customServoRightBack_edit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>170</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="customServoRightFront_edit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>70</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QRadioButton" name="regularDrive_radio">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Regular</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">driveSelGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="tankDrive_radio">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tank</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">driveSelGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="customDrive_radio">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Custom</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">driveSelGroup</string>
    </attribute>
   </widget>
   <widget class="QSlider" name="speedValue_slider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="speedSubmit_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="speed_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Speed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="speedValue_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>-10</y>
      <width>20</width>
      <height>481</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="speed_lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <zorder>customDrive_radio</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>regularDrive_radio</zorder>
   <zorder>tankDrive_radio</zorder>
   <zorder>speedValue_slider</zorder>
   <zorder>speedSubmit_button</zorder>
   <zorder>speed_label</zorder>
   <zorder>speedValue_label</zorder>
   <zorder>line</zorder>
   <zorder>speed_lineEdit</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="tankServoGroup"/>
  <buttongroup name="driveSelGroup"/>
 </buttongroups>
</ui>
